chapters/syntax.tex

```diff
@@ -57,10 +57,8 @@ \section{Lexical conventions}\label{lexical-conventions}
   Modelica uses the same comment syntax as C++ and Java (i.e., \lstinline!//! signals the start of a line comment and \lstinline!/* $\ldots$ */! is a multi-line comment); comments may contain any Unicode character.
   Modelica also has structured comments in the form of annotations and string comments.
 \item
-  Each \lstinline[language=grammar]!description-string! or string in annotations (that is, a \lstinline[language=grammar]!STRING! with production \lstinline[language=grammar]!annotation-clause! in the grammar) may contain any member of the Unicode character set.
-  All other strings have to contain only the subset of Unicode characters identical with the 7-bit US-ASCII character set.
-  \begin{nonnormative}
-  As a consequence, operators like `\lstinline!>!' or `\lstinline!<!', and external functions only operate on ASCII strings and not on Unicode-strings.
+  All Strings may contain any member of the Unicode characters; they are coded as UTF-8 in Modelica files in the file-system.
```

Comment:
A few lines above, we already mention Unicode characters. chapters/packages.tex already mentions UTF-8 is used on the file-system.

Reply:
Yes, had planned to make it a cross-reference - will do that.
I still think that stating that all Unicode characters are allowed is appropriate for the syntax-chapter.

Reply:
Looking more the sentence above makes this part unnecessary; so I remove it. Good catch.